src/Codeception/Lib/InnerBrowser.php

```diff
@@ -1270,14 +1270,15 @@ public function grabValueFrom($field)
         if (!$nodes->count()) {
             throw new ElementNotFound($field, 'Field');
         }
-        return $this->proceedGetValueFromField($nodes);
+        return $this->proceedGetValueFromField($nodes, false);
     }
 
     /**
      * @param Crawler $nodes
+     * @param bool $getOptionTexts
      * @return array|mixed|string
      */
-    protected function proceedGetValueFromField(Crawler $nodes)
+    protected function proceedGetValueFromField(Crawler $nodes, $getOptionTexts)
```

Comment:
I think this should be refactored to offer more general behavior. Maybe to split into 2 methods:

one will return acutal values, second will return content values. 

right now this method does two things and uses hard-to-guess boolean param which changes the behavior.